src/main/java/hw_5/myClass/Point.java

```diff
@@ -0,0 +1,22 @@
+package hw_5.myClass;
+
+public class Point {
+    public int x;
+    public int y;
+
+    public Point(int x, int y) {
+        this.x = x;
+        this.y = y;
+    }
+    public String Info(){
```

Comment:
method name should starts from the small letter

Reply:
fixed